src/model_extensions/use_stock_getter.py

```diff
@@ -1,9 +1,15 @@
-from sodym.classes.mfa_system import MFASystem
-from sodym.classes.stocks_in_mfa import StockWithDSM
-from src.model_extensions.extrapolate_stock import extrapolate_stock
+import logging
+from typing import Dict
+import numpy as np
+from scipy.optimize import least_squares
 
+from sodym import (
+    StockDefinition, Process, Stock, DimensionSet, NamedDimArray, StockWithDSM, StockArray,
+    MFASystem,
+)
 
-class InflowDrivenHistoric_StockDrivenFuture(MFASystem):
+
+class InflowDrivenHistoric_StockDrivenFuture():
```

Comment:
As discussed with Merlin, I'm afraid that has to go back into the MFASystem namespace for the steel model. Sorry about the late notice!

Reply:
This is possibly the best argument for why we should initialise (or have the option to initialise) MFASystem directly with dims, processes, parameters, etc.